### ISSUE TITLE
open parentheses in comments affect indentation

### ISSUE THREAD

**Issue by izm** [millejoh/emacs-ipython-notebook#252]
Example notebook with 2 cells:

In [ ]
```
import pandas as pd

# this is a properly formed comment ()


# this is not properly formed (
    # indents here (
        # indents here
```
        
In [ ]
```
        # this carries forward to more cells
        # until closed )
    # )
# yay

print "what about a string (?"
# still yay
```

Love the package!  Thank you.

**Comment by millejoh** [comment]
This is because ein advises python-indent-calculate-levels and modifies the global variable `python-indent-levels`, which no longer works as that variable has been deprecated for some time. EIN should be advising python-ident-calculate-indentation instead, I think.

**Comment by millejoh** [comment]
Should be fixed with latest commit. Let me know if you are still having problems.

**Comment by izm** [comment]
Hi!

Upgraded to 20171118.924 from Melpa.

Still an issue in `ein:notebook-multilang-mode` (default).  I don't believe it was ever an issue in `ein:notebook-python-mode`.  (Confirmed this using another computer at `ein` 20171020.1157.)

Should I just be using `ein:notebook-python-mode`?

**Comment by millejoh** [comment]
Try calling `M-x eval-expression RET (require 'ein-python)` from the notebook buffer. That seemed to do the trick for me. Might also try deleting all the elc files and then calling `M-x ein:byte-compile-ein`.

**Comment by izm** [comment]
`(require 'ein-python)` changed the auto-indentation a bit, but didn't fix it completely.

Deleting all `elc` files, calling `ein:byte-compile-ein` and restarting emacs did not help.

I have `electric-indent-mode` enabled.

Again, calling `ein:notebook-python-mode` appears to resolve the issue for me.

**Comment by millejoh** [comment]
If you aren't interested in having markdown cells formatted, then using `ein:notebook-python-mode` is fine. Note that you can get mode-specific editing for a cell by calling `C-c C-'` or `M-x ein:edit-cell-contents` while in the cell. This "feature" is mostly a copy of `org-edit-special`.

**Comment by millejoh** [comment]
Finally fixed this one with #270, I hope. Feel free to reopen this if you see this is not the case.
